tests/Integration/Provider/SentinelServiceProviderTest.php

```diff
@@ -11,15 +11,15 @@
  * @see https://github.com/opencfp/opencfp
  */
 
-namespace OpenCFP\Test\Unit\Provider;
```

Comment:
Since this test accesses the application container, it should have been an integration test, I believe.